unfoldingWord/translationCore
# Rebranded 'local user' as 'guest'

#### This pull request addresses:
- Rebranded 'local user' as 'guest'


#### How to test this pull request:
- Try to login as a guest user and make sure all verbiage is updated from 'local user' to guest.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2732)
